### DISCUSSION TITLE
choosing spatial resolution

### DISCUSSION THREAD

**Discussion by agohin** [magpiemodel/magpie#309] [General]
Dear Magpie modellors, 
I am learning your impressive modelling system. At this moment, it is not clear to me if we can access/run the model with different spatial resolution. According to the main publication in GMD 2019, this should be theoretically possible. In practice, I have the impression that we can run with 12 regions and 200 cells. I can imagine that it is too complex to solve for many regions/cells. But can we change the distribution of cells by regions ? Say with more cells for one region (and obviously less for other regions) ? As an example, Europe is currently divided in 10 cells. Would it be possible to introduce more cells (for instance up to the number of EU member states) ? 
Thanks for your advice

**Comment by tscheypidi** [comment]
Changing the spatial resolution is technical possible. The problem is the input data which needs to be available in the chosen resolution. The tools for preparing the data are already available online @ http://github.com/pik-piam (`madrat` and all related `mr`-packages), however some of the original data sources used to produce the aggregated data set cannot yet be automatically downloaded by these packages. 
In practice that means that only the data sets which already have been prepared by us are easily available. We are currently working on solutions to make it easy for everyone to select the resolution in which the model runs, but we are not there yet (if everything works well there might be a solution available by end of the year). 
You can try to create your own input data set in a different resolution, but I would rather recommend to stick for the moment to the available data resolution.

**Reply by agohin** [reply]
Many thanks for the reply and clarifications (just for your info, the link http://github.com/pik-piam/pik-piam does not work). I will check later this year for eventual updates (as well as the data documentation)

**Reply by tscheypidi** [reply]
ah sorry...fixed now the link in my post
